sdk/nodejs/iot/zMixins.ts

```diff
@@ -51,10 +51,10 @@ export class IoTHubEventSubscription extends appservice.EventSubscription<EventH
         // Place the mapping from the well known key name to the Event Hubs account connection string in
         // the 'app settings' object.
 
-        const appSettings = pulumi.all([args.appSettings]).apply(
-            ([appSettings]) => ({ ...appSettings, [bindingConnectionKey]: iotHub.eventHubEventsEndpoint }));
+        const appSettings = pulumi.all([args.appSettings, iotHub.eventHubEventsEndpoint]).apply(
+            ([appSettings, eventHubEventsEndpoint]) => ({ ...appSettings, [bindingConnectionKey]: eventHubEventsEndpoint }));
 
-        super("azure:eventhub:EventHubSubscription", name, bindings, {
+        super("azure:eventhub:IoTHubEventSubscription", name, bindings, {
             ...args,
             resourceGroupName,
             location,
```

Comment:
Nit: an empty line is missing at the end of this file (can't seem to click there...)

Reply:
Fixed in 4befc8cc16a7a663e608029b69f0fd4a8d66b1e2